core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1939,6 +1939,36 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-azurestack
     namespace: ci
+- from:
+    pull-secret:
+      dockerconfigJSON:
+      - auth_field: token_image-puller_app.ci_reg_auth_value.txt
+        item: build_farm
+        registry_url: registry.ci.openshift.org
+      - auth_field: auth
+        email_field: email
+        item: cloud.openshift.com-pull-secret
+        registry_url: cloud.openshift.com
+      - auth_field: auth
+        email_field: email
+        item: quay.io-pull-secret
+        registry_url: quay.io
+      - auth_field: auth
+        email_field: email
+        item: registry.connect.redhat.com-pull-secret
+        registry_url: registry.connect.redhat.com
+      - auth_field: auth
+        email_field: email
+        item: registry.redhat.io-pull-secret
+        registry_url: registry.redhat.io
+      - auth_field: token_image-puller_build02_reg_auth_value.txt
+        item: build_farm
+        registry_url: registry.build02.ci.openshift.org
+  to:
+  - cluster_groups:
```

Comment:
Sounds like we can we append the array item to https://github.com/openshift/release/blob/923a6713e8ba3c4c60bbce2b51ce3d816248cbf1/core-services/ci-secret-bootstrap/_config.yaml#L1897-L1905

Reply:
I copied the content from here. okay, will update to append here.